src/bin/rgb/command.rs

```diff
@@ -667,6 +681,51 @@ impl Command {
                 runtime.accept_transfer(transfer, force)?;
                 eprintln!("Transfer accepted into the stash");
             }
+            Command::SetHost { method, psbt_file } => {
+                let psbt_data = fs::read(&psbt_file)?;
+                let mut psbt = Psbt::deserialize(&psbt_data)?;
+                let mut psbt_modified = false;
+                match method {
+                    CloseMethod::OpretFirst => {
+                        if let Some(outp) = psbt
+                            .unsigned_tx
+                            .output
+                            .iter()
+                            .zip(&mut psbt.outputs)
+                            .find(|(o, outp)| {
+                                o.script_pubkey.is_op_return() && !outp.is_opret_host()
+                            })
+                            .map(|(_, outp)| outp)
+                        {
+                            psbt_modified = true;
+                            outp.set_opret_host().ok();
+                        }
```

Comment:
This is nit: we can optimize it even further

```suggestion
                        psbt
                            .unsigned_tx
                            .output
                            .iter()
                            .zip(&mut psbt.outputs)
                            .find(|(o, outp)| {
                                o.script_pubkey.is_op_return() && !outp.is_opret_host()
                            })
                            .and_then(|(_, outp)| {
                                psbt_modified = true;
                                outp.set_opret_host()
                            })
```

Reply:
idem